alignments/rec/Brick-REC-nodevice.rdf

```diff
@@ -6,7 +6,7 @@
      xmlns:xml="http://www.w3.org/XML/1998/namespace"

```

Comment:
Is there a specific reason that REC alignment should be in rdf-xml instead of ttl?

Reply:
I'd prefer using a single format across all the ontologies within the repo when it's possible.

Reply:
That's the form of the file that we received from the REC team. I just left it in that form so that it would be easier to incorporate future changes/alignments if/when those occur

Reply:
But it's weird that we have multiple alignment files for the same ontology in different formats (https://github.com/BrickSchema/Brick/blob/efbbff7353cddae6f9c180d907d2bd02e8f084e2/alignments/rec/Brick-REC-alignment.ttl)

And they don't keep rdf-xml in alignments anyway: https://github.com/RealEstateCore/rec/tree/master/ontology/alignments

I'm fine either way, and if you still feel the rdf-xml is better, please resolve this conversation.